scipy/stats/_discrete_distns.py

```diff
@@ -937,6 +937,21 @@ def _stats(self, a):
     def _entropy(self, a):
         return a / sinh(a) - log(tanh(a/2.0))
 
+    def _rvs(self, a):
+        # The discrete Laplace is equivalent to the two-sided geometric
+        # distribution with PMF:
+        #   f(k) = (1 - alpha)/(1 + alpha) * alpha^abs(k)
+        # Furthermore, the two-sided geometric distribution is
+        # equivalent to the difference between two iid geometric 
+        # distributions.
+        # Thus, we can leverage the following:
+        #   1) alpha = e^-a
+        #   2) probability_of_success = 1 - alpha (Bernoulli trial)
+        probOfSuccess = 1. - np.exp(-np.asarray(a))
```

Comment:
-expm1(-a) is likely more stable in the small `a` limit

Reply:
Awesome! I should be able to update this by tomorrow, at the latest.

Reply:
Complete. Thanks for the recommendation!